### ISSUE TITLE
Publish with external Time Slot Mode Error

### ISSUE THREAD

**Issue by m4chine23** [SergeyMosin/Appointments#563]
When I publish the Site with external Time Slot Mode a Internal Server Error. Changing to Simple or Weekly with same Calendars works.

**Comment by m4chine23** [comment]
Appointments Version: 2.3.0
NC-AIO

```
[index] Fehler: strpos(): Argument #1 ($haystack) must be of type string, resource given in file '/var/www/html/custom_apps/appointments/lib/Backend/BCSabreImpl.php' line 192
	GET /apps/appointments/form?p=p0&v=1732374754822
	von xx.xxx.xxx.xx von HHC um 23.11.2024, 16:12:35
```

**Comment by SergeyMosin** [comment]
Thanks. I think this is might be because of Postgresql vs MySQL blob handling. The fix will be released asap.

**Comment by SergeyMosin** [comment]
fixed in v2.3.1

**Comment by m4chine23** [comment]
it works! Thank you Sir!
